modules/pam_keyinit/pam_keyinit.c

```diff
@@ -234,6 +230,31 @@ int pam_sm_open_session(pam_handle_t *pamh, int flags UNUSED,
 	return ret;
 }
 
+/*
+ * Dummy
+ */
+int pam_sm_authenticate(pam_handle_t *pamh UNUSED, int flags UNUSED,
+                   int argc UNUSED, const char **argv UNUSED)
+{
+  return PAM_SUCCESS;
```

Comment:
Here PAM_IGNORE should be returned to make it similar to the other setcred-only modules.